### ISSUE TITLE
Some letters of line do not render until selected manually

### ISSUE THREAD

**Issue by Saphereye** [neovide/neovide#2423]
**Describe the bug**
On opening a new file few letters of some words are not rendered, until manually selected.

**To Reproduce**
1. ~~Install  [NvChad](https://github.com/NvChad/NvChad) (Unsure about its relevance but I am using this plugin)~~ (@Qwerty-Space is having same issue although NvChad is not installed)
1. Open a few files

**Expected behavior**
No letters should disappear until explicitly requested.

**Screenshots**
Here the first letter of `let`, `Fibonacci` and `Random` in `distribution.rs` are not visible until selected manually. But interestingly `main.rs`  has all letters rendered.
[Screencast from 17-03-24 04:10:55 PM IST.webm](https://github.com/neovide/neovide/assets/59739923/e3649c52-3c93-44f6-94be-5bdfd9ecb5d5)

**Desktop:**
 - OS: Ubuntu 22.04.4 LTS x86_64
 - Neovide Version 0.11.1
 - Neovim Version v0.10.0-dev
 - LuaJIT 2.1.0-beta3
 - Build type: RelWithDebInfo

**Logs**
[neovide_r2024-03-17_16-13-51.restart-0000.log](https://github.com/neovide/neovide/files/14627102/neovide_r2024-03-17_16-13-51.restart-0000.log)
[neovide_rCURRENT.log](https://github.com/neovide/neovide/files/14627103/neovide_rCURRENT.log)

**Additional context**
My `init.lua` file:
```lua
require "core"

local custom_init_path = vim.api.nvim_get_runtime_file("lua/custom/init.lua", false)[1]

if custom_init_path then
  dofile(custom_init_path)
end

require("core.utils").load_mappings()

local lazypath = vim.fn.stdpath "data" .. "/lazy/lazy.nvim"

-- bootstrap lazy.nvim!
if not vim.loop.fs_stat(lazypath) then
  require("core.bootstrap").gen_chadrc_template()
  require("core.bootstrap").lazy(lazypath)
end

dofile(vim.g.base46_cache .. "defaults")
vim.opt.rtp:prepend(lazypath)
require "plugins"

-- Set vfx mode for neovide to railgun
vim.g.neovide_cursor_vfx_mode = "railgun"

-- Set neovide refresh rate
vim.g.neovide_refresh_rate = 60
```

I appreciate your time and effort in looking into this matter

**Comment by Qwerty-Space** [comment]
I'm having the same issue, but I do not have NvChad installed.

**Comment by fredizzimo** [comment]
I'm pretty sure that this was fixed by:
* https://github.com/neovide/neovide/pull/2012 

And released in Neovide 0.11.2.

There was also another similar fix released in 0.13.0:
* https://github.com/neovide/neovide/pull/2298
